modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesKernelBoot.java

```diff
@@ -72,23 +72,25 @@
     /**
      * This will boot up all the components in Andes kernel and bring the server to working state
      */
-    public static void bootAndesKernel() throws AndesException, UnknownHostException, JMException {
-
-        isKernelShuttingDown = false;
-        //loadConfigurations - done from outside
-        //startAndesStores - done from outside
-        int threadPoolCount = 1;
-        andesRecoveryTaskScheduler = Executors.newScheduledThreadPool(threadPoolCount);
-        startAndesComponents();
-        startHouseKeepingThreads();
-        syncNodeWithClusterState();
-        registerMBeans();
-        startThriftServer();
-        startMessaging();
-        createSuperTenantDLC();
-
-        Andes.getInstance().startSafeZoneAnalysisWorker();
-
+    public static void bootAndesKernel() throws AndesException {
+        try {
+            isKernelShuttingDown = false;
+            //loadConfigurations - done from outside
+            //startAndesStores - done from outside
+            int threadPoolCount = 1;
+            andesRecoveryTaskScheduler = Executors.newScheduledThreadPool(threadPoolCount);
+            startAndesComponents();
+            startHouseKeepingThreads();
+            syncNodeWithClusterState();
+            registerMBeans();
+            startThriftServer();
+            startMessaging();
+            createSuperTenantDLC();
+
+            Andes.getInstance().startSafeZoneAnalysisWorker();
+        } catch (JMException e) {
```

Comment:
If any exception was thrown here, that means some part of the broker is not working. Is catching JMS exception is enough? If some error happened here, it might be wrong to start the broker without fatal warnings?